apps/meteor/client/lib/voip/VoIPUser.ts

```diff
@@ -156,28 +168,29 @@ export class VoIPUser extends Emitter<VoipEvents> {
 		/**
 		 * Re-registration post network recovery should be attempted
 		 * if it was previously registered or incall/onhold
-		 * */
+		 */
 
 		if (this.registerer && this.callState !== 'INITIAL') {
-			this.attemptRegistrationPostRecovery();
+			this.attemptRegistration = true;
 		}
 	}
 
-	onDisconnected(error: any): void {
+	onDisconnected(_error: any): void {
```

Comment:
Using _ on param names is for when you want to ignore the param :)

Reply:
Fixed.